addons/website_crm_partner_assign/views/website_crm_partner_assign_templates.xml

```diff
@@ -19,153 +17,241 @@
 
 <template id="index" name="Find Resellers">
     <t t-call="website_crm_partner_assign.layout">
-        <div class="col-lg-12">
-            <h1 class="text-center">
-                Looking For a Local Store?
-            </h1><h2 class="text-center text-muted">
-                Contact a reseller
-            </h2>
-        </div>
-
-        <div class="col-lg-3 mb32" id="partner_left">
-
-            <ul id="reseller_grades" class="nav nav-pills flex-column mt16">
-                <li class="nav-header nav-item"><h3>Filter by Level</h3></li>
-                <t t-foreach="grades" t-as="grade">
-                    <li class="nav-item">
+        <div class="o_wcrm_filters_top d-flex justify-content-end flex-wrap gap-2 w-100 d-print-none">
+        <!-- Is there a way to retrieve the active country without looping ? -->
+            <t t-foreach="countries" t-as="country">
+                <h4 t-if="country['active']" class="my-0 me-auto pe-sm-4 lh-base">Find a reseller <span t-if="country['country_id'][0] != 0">in <t t-out="country['country_id'][1]"></t></span></h4>
+            </t>
+            <div id="o_wcrm_reseller_grades" class="dropdown d-none d-lg-block">
+                <a role="button" href="#" data-bs-toggle="dropdown" class="dropdown-toggle btn btn-light" aria-expanded="true" aria-label="Open categories dropdown">
+                    <t t-foreach="grades" t-as="grade">
+                        <span t-if="grade['active']" t-out="grade['grade_id'][1]"/>
+                    </t>
+                </a>
+                <div class="dropdown-menu">
+                    <t t-foreach="grades" t-as="grade">
                         <a t-attf-href="/partners#{ grade['grade_id'][0] and '/grade/%s' % grade['grade_id'][0] or '' }#{ current_country and '/country/%s' % slug(current_country) or '' }#{ '?' + (search_path or '') + '&amp;' + keep_query('country_all') }"
-                           t-attf-class="nav-link#{grade['active'] and ' active' or ''}">
-                            <span class="badge rounded-pill float-end" t-esc="grade['grade_id_count'] or ''"/>
-                            <t t-esc="grade['grade_id'][1]"/>
-                        </a>
-                    </li>
-                </t>
-            </ul>
-
-            <ul id="reseller_countries" class="nav nav-pills flex-column mt16">
-                <li class="nav-header nav-item"><h3>Filter by Country</h3></li>
-                <t t-foreach="countries" t-as="country">
-                    <li t-if="country['country_id']" class="nav-item">
+                        class="dropdown-item" t-out="grade['grade_id'][1]"/>
+                    </t>
+                </div>
+            </div>
+            <div id="o_wcrm_reseller_countries" class="dropdown d-none d-lg-block">
+                <a role="button" href="#" data-bs-toggle="dropdown" class="dropdown-toggle btn btn-light" aria-expanded="true" aria-label="Open countries dropdown">
```

Comment:
Searching while having a country selected will "hide" the dropdown label (the same for "categories" actually). 
![image](https://user-images.githubusercontent.com/8558351/231213080-f4788721-fa69-4fb9-9c61-dff14d689879.png)